caffe2/onnx/backend.cc

```diff
@@ -971,24 +972,27 @@ Caffe2Ops Caffe2Backend::ConvertNode(const std::string& node_str, int opset_vers
   return OnnxNodeToCaffe2Ops(init_model, pred_model, &onnx_node, opset_version);
 }
 
-Caffe2Ops Caffe2Backend::OnnxNodeToCaffe2Ops(const ModelProto &init_model,
-                                            const ModelProto &pred_model,
-                                            OnnxNode* onnx_node,
-                                            int opset_version) {
+Caffe2Ops Caffe2Backend::OnnxNodeToCaffe2Ops(
+    const ModelProto& init_model,
```

Comment:
Do we really need init_model and pred_model here? I guess later RNN may need, probably for now, we can remove it, Since you don't even use it.

Reply:
For now we don't but generally speaking if we want to support RNN ops, we need it.